docs/reference/01-overview.md

```diff
@@ -78,4 +78,18 @@ contracts (such as MerkleLockupFactory, MerkleLL and MerkleLT).
 
 ## Flow
 
-Coming soon.
+> [**Flow Source Code**](https://github.com/sablier-labs/flow/tree/v1.0.0)
+
+Flow consists of SablierFlow, and an NFT descriptor.
+
+### NFTDescriptor
+
+> [**FlowNFTDescriptor Reference**](./flow/contracts/contract.FlowNFTDescriptor)
+
+Generates the URI describing the Sablier Flow stream NFTs.
```

Comment:
let's mention that it currently shows the logo

Reply:
https://github.com/sablier-labs/docs/pull/214/commits/65c2f4f707eba738ac4902c089e4106cdac6cf50